pegasus/sites.v3/code.org/public/js/workshop-search.js

```diff
@@ -23,83 +25,89 @@ function initializeMap() {
 }
 
 function loadWorkshops() {
-  var params = [];
+  markerClusterer = new MarkerClusterer(gmap, [],
+    {imagePath: 'https://developers.google.com/maps/documentation/javascript/examples/markerclusterer/m'}
```

Comment:
On that note, this might be a good time to move this js over to our js pipeline under `apps` and leverage the [npm module](https://www.npmjs.com/package/js-marker-clusterer).